Configs/.config/fish/config.fish

```diff
@@ -4,9 +4,22 @@ if status is-interactive
     starship init fish | source
 end
 
+# List Directory
 alias ls="lsd"
 alias l="ls -l"
 alias la="ls -a"
 alias lla="ls -la"
 alias lt="ls --tree"
 
+# Handy change dir shortcuts
+abbr .. 'cd ..'
+abbr ... 'cd ../..'
```

Comment:
Thank you, for your support and contribution!
This is same as the next line, looks like a typo.
Maybe we can use `.2` and `.3` to distinguish it?

Reply:
no that's intentional, i left it at .. and ... because tapping the same .. twice is just faster to me. .2 and .3 was how I was doing it before when i originally found the abbrs from Derek's video. i quickly realized it required pressing the two different keys vastly apart from eachother which takes away the benefit it adds.
Try the current setup as is, you'll very quickly get used to it.

Reply:
i did the .4 and .5 are just there for sanity's sake, if you need to go up so many dirs smashing "....." was a little weird because the ligatures brought the periods so close it was hard to read

Reply:
I meant these two lines have the same abbr `...` -

```
abbr ... 'cd ../..'
abbr ... 'cd ../../..'
```

both have the same abbr so the 'cd ../..' doesn't work as the second line overrides the 1st.